src/aframe-streetmix-parsers.js

```diff
@@ -524,9 +524,28 @@ function createSegmentElement (scaleX, positionX, positionY, rotationY, mixinId,
   segmentEl.setAttribute('scale', scaleNew);
   // segmentEl.setAttribute('geometry', 'height', length); // alternative to modifying scale
   segmentEl.setAttribute('position', positionX + ' ' + positionY + ' 0');
-
   segmentEl.setAttribute('rotation', '270 ' + rotationY + ' 0');
   segmentEl.setAttribute('mixin', mixinId);
+
+  //TODO: create a new mixin object that acts as a curb
+  if (mixinId == 'sidewalk'){
+    var leftCurb = document.createElement('a-entity');
+    leftCurb.setAttribute('id', "left curb");
+    //-(scaleX*3*3.28084)*0.5
+    leftCurb.setAttribute('position', 'x', -(scaleX*3*0.5));
```

Comment:
`    leftCurb.setAttribute('position', 'x', -(segmentWidthInMeters / 2)); `

`    rightCurb.setAttribute('position', 'x', (segmentWidthInMeters / 2)); `

The `segmentWidthInMeters` variable for the current segment will need to be passed to this function

Reply:
Hi @kfarr. I tried this and it did not resolve the issue.